addons/ai/fnc_waypointGarrison.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 /* ----------------------------------------------------------------------------
-Script: fnc_waypointGarrison
+Script: CBA_fnc_waypointGarrison
```

Comment:
How many times do I have to reject this? It is not a function, but compiled as script by a waypoint via config entry. 

`CBA_fnc_waypointGarrison` does not exist. fnc_waypointGarrison.sqf does and it is only meant to be used as uncompiled script.

Reply:
do we want to hide that from the Docs then? or what to do with it?

Reply:
Hide it if you want, but it's no big deal. Do not name it "`CBA_fnc_waypointGarrison`" though, because that is not a thing.